docs/cheaha/slurm/slurm_tutorial.md

```diff
@@ -0,0 +1,528 @@
+# Writing Slurm Batch Jobs
```

Comment:
I don't want to hold this one up any longer, so this doesn't have to be done as part of this review, it's a suggestion for the future.

I want to add a part at the beginning of the tutorial that tells people when/why they would want to use each type, so let's create an issue for it.

Something like:

```
## How do I know which one I need?

- Use 1 if...
- Use 2 if...
```

Reply:
Sure. I will submit an issue.